Jenkinsfile

```
@@ -0,0 +1,71 @@
+pipeline {
+  agent { label 'linux' }
+
+  options {
+    disableConcurrentBuilds()
+    /* checkout logos repo to use site builder */
```

Same here. This is not a useful comment. Do not describe WHAT code does, describe WHY it does it.
```suggestion
    /* Necessary for logos-side-builder local_folder source type. */
```